src/TestSuite/Fixture/TestFixture.php

```diff
@@ -139,6 +139,7 @@ protected function _schemaFromReflection(): void
         try {
             $name = Inflector::camelize($this->table);
             $ormTable = $this->fetchTable($name, ['connection' => $db]);
+            $this->getTableLocator()->remove($name);
```

Comment:
Can you add a simple one-line comment above this documenting that it's intentional to avoid an issue.  We'll probably clean TestFixture up further in 5.x.

Reply:
Best I could do were two lines.